ingestion/src/metadata/cmd.py

```diff
@@ -114,8 +115,10 @@ def profile(config: str) -> None:
     try:
         logger.debug(f"Using config: {workflow_config}")
         workflow = ProfilerWorkflow.create(workflow_config)
-    except ValidationError as e:
-        click.echo(e, err=True)
+    except ValidationError as err:
+        logger.debug(traceback.format_exc())
```

Comment:
@nahuelverdugo even when I enable debug the traceback wasn't getting printed. Lot of times I needed to change the logger.debug to print to get the full trace. Can you check why that may be happening

Reply:
I will check that before merging

Reply:
It is working fine. The only problem is that we call `set_loggers_level` in `logger.py` from different places. I will create a new issue for reviewing that.